src/Backend/API.ts

```diff
@@ -29,20 +29,33 @@ interface BackendMap {
   [key: string]: Backend;
 }
 
+// TODO Move outside API.ts
 // List of backend extensions registered
 let globalBackendMap: BackendMap = {};
 // List of Executor extensions registered
 let globalExecutorArray: Executor[] = [];
 
 const logTag = "API";
 
+// TODO Move outside API.ts
+export namespace BackendContext {
+  export const isRegistered = (backendName: string) => {
+    return globalBackendMap[backendName] !== undefined;
+  };
+}
+
 const registerBackend = (backend: Backend) => {
   const backendName = backend.name();
   assert(backendName.length > 0);
   globalBackendMap[backendName] = backend;
   const compiler = backend.compiler();
   if (compiler) {
     gToolchainEnvMap[backend.name()] = new ToolchainEnv(compiler);
+    vscode.commands.executeCommand(
```

Comment:
FYI, this is the context that can be referred in package.json

Reply:
There is no `getContext` API in vscode. Therefore, both apis should be provided to get the backend status.